CONTRIBUTING.md

```diff
@@ -1 +1,78 @@
 # Contributing to Chef Server
+
+Thank you for your interest in Chef Server!  We use **GitHub Issues**
+for issue tracking and contributions:
+
+1. Reporting an issue or making a feature request
+2. Submitting patches for new features or bug fixes
+
+## Submitting Patches
+
+All patches should be submitted as GitHub pull requests:
+
+1. Sign or be added to an existing
+[Contributor License Agreement (CLA)](https://supermarket.getchef.com/become-a-contributor).
+
+2. Create a GitHub Pull Request.
+
+3. If you are a Chef Software employee, submit an ad-hoc
+chef-server-12 build for your branch and post a link to in the pull
+request. For non Chef Software employees, your job will be submitted
+for you by one of the reviewers.
+
+4. Respond [Code Review](#cr) to code review comments to improve the
+code until it is ready to merge.
+
+5. Merge it! (If you don't have commit access, a maintainer will merge
+   it for you after code review is complete).
+
+## Reporting an Issue
+
+When reporting an issue, please try to include as much detail as
+possible, including:
+
+- The version of the Chef Server you are running,
+- The operating system you are running it on,
+- What you are trying to achieve but can't,
+- The steps to reproduce the issue you are seeing,
+- What you expect to see given the steps you've described, and
+- What you actually see given the steps you've described.
+
+## <a name="cr"></a> Code Review Process
+
+The Chef Code Review process happens on Github pull requests. See
+[this article](https://help.github.com/articles/using-pull-requests)
+if you're not familiar with Github Pull Requests.
+
+Once you a pull request, the **Chef Engineering Team** or **Chef Core
+Committers** will review your code and respond to you with any
```

Comment:
Is there a maintainer team for Chef Server that isn't "Chef Core Committers"? Does chef-core even have commit access here?